sphinx_rtd_theme/breadcrumbs.html

```diff
@@ -22,7 +22,7 @@
 <div role="navigation" aria-label="{{ _('Page navigation') }}">
   <ul class="wy-breadcrumbs">
     {%- block breadcrumbs %}
-      <li><a href="{{ pathto(master_doc) }}" class="icon icon-home"></a></li>
+      <li><a href="{{ pathto(master_doc) }}" class="icon icon-home"><span class="visuallyhidden">Home</span></a></li>
```

Comment:
I'm guessing we probably want `aria-hidden="true"` on the icon still, and I would normally use `aria-label` here instead of a nested element with the text. What is the case for using the hidden `span` instead of `aria-label`? The interactive element example describe how I'd normally address this:

![image](https://user-images.githubusercontent.com/1140183/203176325-0791e1d4-338d-453e-871d-89a008db25c5.png)

Reply:
Good point. Replaced it with aria-label instead. Force of habit with icons.